TimeTrackerClient/TimeTrackerClient/Views/Login/LoginView.swift

```diff
@@ -11,78 +11,50 @@ struct LoginView: View {
     @ObservedObject private(set) var viewModel: LoginViewModel
     @ObservedObject var keyboardResponder = KeyboardResponder()
```

Comment:
hai sa mai facem ceva aici legat de Keyboard. Cand esti pe primul field, sa ai pe tastatura `Next` in loc de `Return` si cand il apesti sa te duca pe Password. Iar cand esti pe password, sa ai `Done` sau `Send` si cand il apesti fie iti face dismiss, fie iti face login (daca e send). Si inca o chestie, sa poti sa ii dai dismiss fie cand faci swipe in jos, fie cand dai tap pe langa